pom.xml

```diff
@@ -30,7 +30,7 @@
 		<logback.version>1.1.11</logback.version>
 		<selenium.version>3.0.0</selenium.version>
 		<build.server.version>1.2.0</build.server.version>
-		<git.server.version>1.0.10</git.server.version>
+		<git.server.version>1.0.11-SNAPSHOT</git.server.version>

```

Comment:
I think this is causing your Travis CI error. Look at the error message:

`[ERROR] Failed to execute goal on project devhub: Could not resolve dependencies for project nl.devhub:devhub:jar:1.0-SNAPSHOT: Failed to collect dependencies at nl.tudelft.ewi.git:git-models:jar:1.0.11-SNAPSHOT: Failed to read artifact descriptor for nl.tudelft.ewi.git:git-models:jar:1.0.11-SNAPSHOT: Could not transfer artifact nl.tudelft.ewi.git:git-models:pom:1.0.11-SNAPSHOT from/to codehaus-snapshots (https://nexus.codehaus.org/snapshots/): nexus.codehaus.org: Unknown host nexus.codehaus.org -> [Help 1]`